tools/sign_release.sh

```diff
@@ -87,8 +71,18 @@ curl -F "input=@$BUILD_DIR/app-amazonWebview-release-unsigned.apk" \
 zipalign -v 4 $BUILD_DIR/$FINAL_NAME $BUILD_DIR/app-amazonWebview-release-aligned.apk
 mv -f $BUILD_DIR/app-amazonWebview-release-aligned.apk $BUILD_DIR/$FINAL_NAME
 
+
+# We don't use `-Werr` (treat warnings as errors) because errors related to certain AndroidX
+# files are expected here.
+#
+# When the build is signed by Autograph, files inside the META-INF folder are not signed (due to
+# an outdated dependency on their end). We previously stripped these files out of the APK to
+# avoid warnings, but after migrating to AndroidX important files are now stored in this
+# directory (they appear to be Jetified libraries, but we haven't verified that). As we can't
+# strip these files out, we stop failing on warnings instead.
```

Comment:
nit: can we file a bug to re-enable -Werr when autograph does the right thing and link to the issue number here?

Reply:
Opened https://github.com/mozilla-mobile/firefox-echo-show/issues/290